src/VolleyManagement.Backend/VolleyManagement.Services/GameService.cs

```diff
@@ -144,9 +144,9 @@ public GameResultDto Get(int id)
         /// </summary>
         /// <param name="tournamentId">Identifier of the tournament.</param>
         /// <returns>List of game results of specified tournament.</returns>
-        public List<GameResultDto> GetTournamentResults(int tournamentId)
+        public ICollection<GameResultDto> GetTournamentResults(int tournamentId)
         {
-            var allGames = QueryAllTournamentGames(tournamentId)
+            var allGames = ((List<GameResultDto>)(QueryAllTournamentGames(tournamentId)))
```

Comment:
Why cast?

Reply:
becouse QueryAllTournamentGames retrns ICollection<T> from _tournamentGameResultsQuery which is from IQuery<ICollection<GameResultDto>, TournamentGameResultsCriteria>  which is required to be ICollection

Reply:
cast is necessary on ICollection does`t support FindAll()

Reply:
find another way

Reply:
ToList() ?